libs/blocks/marquee/marquee.css

```
@@ -180,9 +180,9 @@
   margin-bottom: var(--spacing-s);
 }
 
-.marquee .text .icon-area picture {
-  display: block;
-  line-height: 0;
+.marquee .text .icon-area picture,
+.marquee .text .icon-area a {
```

Don't we get this regardless of the `<a>` with the `<picture>`?  This runs twice now when there is `<a>`.

I think you could simplify this a bit and still work as expected:
```suggestion
.marquee .icon-area picture {
```

---

without both 
```
.marquee .icon-area picture,
.marquee .icon-area a {
```
the one-icon link ends up [taking up the full width of the text area](https://main--cc--adobecom.hlx.page/drafts/solene/mwpw-128103/svg-with-link?martech=off&milolibs=mwpw-128103)